R/pkg/vignettes/sparkr-vignettes.Rmd

```diff
@@ -1,12 +1,13 @@
 ---
 title: "SparkR - Practical Guide"
 output:
-  html_document:
-    theme: united
+  rmarkdown::html_vignette:
     toc: true
     toc_depth: 4
-    toc_float: true
-    highlight: textmate
+vignette: >
```

Comment:
So I think the `theme` was the one giving us the nice looking HTML . Is that not supported with `rmarkdown::html_vignette` ?